general-knowledge/about-mycrypto/official-domain-names-used-by-mycrypto.md

```diff
@@ -3,7 +3,7 @@ title: Official Domain Names Used By MyCrypto
 description: Domains owned by MyCrypto
 priority: 100
```

Comment:
The priority could probably be lower, since this won't be relevant for most of the users. Personally I'd say move it to the bottom (priority < 40).

Reply:
Updated it to 40 in 36eebcc653efa61a5ee9714047320427c59e36b7